WooCommerce/WooCommerceTests/ViewRelated/Products/SKU Scanner/BarcodeSKUScannerProductFinderTests.swift

```diff
@@ -3,8 +3,8 @@ import XCTest
 import Yosemite
 @testable import WooCommerce
 
-final class BarcodeSKUScannerProductFinderTests: XCTestCase {
-    private var sut: BarcodeSKUScannerProductFinder!
+final class BarcodeSKUScannerItemFinderTests: XCTestCase {
+    private var sut: BarcodeSKUScannerItemFinder!
     var stores: MockStoresManager!
     var storageManager: MockStorageManager!
     var analyticsProvider: MockAnalyticsProvider!
```

Comment:
I see these are outside of the PR changes, but we can make their access control private as `BarcodeSKUScannerProductFinder` :D 
```suggestion
    private var stores: MockStoresManager!
    private var storageManager: MockStorageManager!
    private var analyticsProvider: MockAnalyticsProvider!
    private var analytics: WooAnalytics!
```